suites/reef/rgw/tier-2_rgw_ms-archive.yaml

```diff
@@ -248,311 +253,104 @@ tests:
-              - "ceph config set client.rgw.{daemon_id:shared.pri} rgw_crypt_sse_s3_vault_secret_engine transit"
-              - "radosgw-admin zone placement modify --rgw-zone primary --placement-id default-placement  --compression zlib"
-              - "radosgw-admin period update --commit"
-              - "ceph orch restart {service_name:shared.pri}"
-        ceph-arc:
-          config:
-            cephadm: true
-            commands:
-              - "ceph config set client.rgw.{daemon_id:shared.arc} rgw_crypt_require_ssl false"
-              - "ceph config set client.rgw.{daemon_id:shared.arc} rgw_crypt_sse_s3_backend vault"
-              - "ceph config set client.rgw.{daemon_id:shared.arc} rgw_crypt_sse_s3_vault_addr http://127.0.0.1:8100"
-              - "ceph config set client.rgw.{daemon_id:shared.arc} rgw_crypt_sse_s3_vault_auth agent"
-              - "ceph config set client.rgw.{daemon_id:shared.arc} rgw_crypt_sse_s3_vault_prefix /v1/transit "
-              - "ceph config set client.rgw.{daemon_id:shared.arc} rgw_crypt_sse_s3_vault_secret_engine transit"
-              - "radosgw-admin zone placement modify --rgw-zone archive --placement-id default-placement  --compression zlib"
-              - "radosgw-admin period update --commit"
-              - "ceph orch restart {service_name:shared.arc}"
-      desc: Setting vault configs for sse-s3 on multisite
-      module: exec.py
-      name: sse-s3 vault configs, and zlib compression
-      polarian-id: CEPH-83575916
-
-
+            extra-pkgs:
+              - wget https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.rpm
+            install_start_kafka_archive: true
+            script-name: test_bucket_notifications.py
+            config-file-name: test_bucket_notification_kafka_broker_ms_replication_from_pri.yaml
   - test:
-      clusters:
-        ceph-pri:
-          config:
-            script-name: test_sse_s3_kms_with_vault.py
-            config-file-name: test_sse_s3_bucket_enc_multipart_download_remote_site.yaml
-      desc: test multipart download at remote site
-      polarion-id: CEPH-11357
+      name: notify on multisite replication delete events with kafka_broker on arc site
+      desc: notify on multisite replication delete events with kafka_broker on arc site
+      polarion-id: CEPH-83575922
       module: sanity_rgw_multisite.py
-      name: test multipart download at remote site
-# test the workaround to sync only from one active zone
-  - test:
       clusters:
         ceph-arc:
           config:
-            cephadm: true
-            commands:
-              - "radosgw-admin zone modify --rgw-zone archive --sync_from primary --sync_from_all false"
-              - "radosgw-admin period update --commit"
-              - "radosgw-admin period get"
-      desc: test the workaround to sync only from one active zone
-      module: exec.py
-      name: test the workaround to sync only from one active zone
-      polarian-id: CEPH-83581371
```

Comment:
polarion id is missing, but looks like step covered in setup ms.

Reply:
yes